pkg/operator/controller/controller.go

```diff
@@ -184,20 +164,14 @@ func (r *reconciler) reconcile() (reconcile.Result, error) {
 		// Handle active ingress.
 		err := r.ensureRouterForIngress(&ingress, dnsConfig, infraConfig)
 		if err != nil {
-			errs = append(errs, fmt.Errorf("failed to ensure clusteringress %s/%s: %v", ingress.Namespace, ingress.Name, err))
+			errs = append(errs, err)
 		}
 	}
 
+	// Only ensure the router-ca configmap if there have been no errors up
+	// to now.
```

Comment:
Why not keep going? What if there's a functional clusteringress that could be making use of the CA despite another ingress having trouble? What if the error actually had nothing to do with the ingress specifically and everything's actually fine? Whether to create the CA seems to be a function of the *existence* of clusteringresses, so I'd expect `ensureRouterCAConfigmap` to go ahead and ensure the CA if any ingresses are in the list.

Reply:
It could prematurely delete the configmap if updating a clusteringress with a custom certificate fails.  However, we are better off having that edge case than having the rather common case that something fails repeatedly and the configmap never gets created, so I'll change it.

Reply:
Fixed.  Thanks!